src/containers/LoginView.jsx

```diff
@@ -38,7 +38,7 @@ class LoginView extends React.Component {
     return (
       <div className="tile is-child box">
         <h1 className="title">Login</h1>
-        <form onSubmit={this.loginUser}>
+        <form>
```

Comment:
Why did you change this?

Reply:
it doesn't do anything

Reply:
I think leaving it out means that the Enter/Return key doesn't work to submit the form. Can you try both and confirm?

Reply:
yep still works